docs/installation/ubuntulinux.md

```diff
@@ -146,7 +146,7 @@ To upgrade your kernel and install the additional packages, do the following:
 
 5. After your system reboots, go ahead and install Docker.
 
-
+>**Note**:  In order for Docker to start on Ubuntu 14.04 or lower, apparmor needs to be installed. It can be downloaded using the following command: `apt-get install apparmor`
```

Comment:
@thaJeztah  You are correct. This information should be a paragraph in the prerequisites not a a note at the end of the procedure.  Just add this para to the pre

```markdown
If you are installing on Ubuntu 14.04 or 12.04, `apparmor` is required.  You can install it using: `apt-get install apparmor`
```